cli/cephadm/cephadm.py

```diff
@@ -5,11 +5,13 @@
 class CephAdm(Cli):
     """This module provides CLI interface to manage the CephAdm operations."""
 
-    def __init__(self, nodes, base_cmd="cephadm"):
+    def __init__(self, nodes, mount=None, base_cmd="cephadm"):
         super(CephAdm, self).__init__(nodes)
 
         self.base_cmd = base_cmd
         self.base_shell_cmd = f"{self.base_cmd} shell"
+        if mount:
+            self.base_shell_cmd += f" --mount {mount} --"
```

Comment:
Do we need `--` at the end ?

Reply:
@vamahaja 
Yes since ceph orch command separated by "--" with mount command
command : cephadm shell -- mount /tmp:/tmp -- ceph orch tuned-profile apply -i <spec_file>